### ISSUE TITLE
(CSS): Escaped identifiers are re-escaped when they shouldn't be

### ISSUE THREAD

**Issue by jafin** [trullock/NUglify#379]
**Describe the bug**

`Uglify.Css` is escaping already escaped characters in class identifiers
The example below is output from tailwind build.

**To Reproduce**

```c#
var result = Uglify.Css(@"
.\!inline-flex {
    display: inline-flex!important
}
");
Console.WriteLine(result.Code);
```

### Actual

```css
.\\21inline-flex {
    display: inline-flex!important
}
```

### Expected

```css
.\!inline-flex {
    display: inline-flex!important
}
```

I've done up a PR but I have been unable to run the tests locally (Am not setup for net4.6) ,

**Comment by trullock** [comment]
Fixed in v1.21.3

**Comment by jafin** [comment]
Thank you!
